app/src/main/java/social/entourage/android/onboarding/OnboardingEmailPwdFragment.kt

```diff
@@ -40,12 +40,11 @@ class OnboardingEmailPwdFragment : Fragment() {
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
 
-        if (tempEmail == null || !tempEmail!!.isValidEmail()) {
-            callback?.updateButtonNext(false)
-        }
-        else {
+        if (tempEmail.isValidEmail()) {
```

Comment:
tempEmail can't be null ?

Reply:
tempEmail can be null because it comes from fragment arguments which can be null
I have added the nullability check to isValidEmail extension because I have met this situation several times in the project (=> no need to check it here, a null email is not valid)